{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "CloudForms Management Engine 5",
      "fix_state": "Not affected",
      "package_name": "ansible",
      "cpe": "cpe:/a:redhat:cloudforms_managementengine:5"
    },
    {
      "product_name": "Red Hat Ansible Engine 2",
      "fix_state": "Not affected",
      "package_name": "ansible",
      "cpe": "cpe:/a:redhat:ansible_engine:2"
    },
    {
      "product_name": "Red Hat Ansible Tower 3",
      "fix_state": "Not affected",
      "package_name": "ansible",
      "cpe": "cpe:/a:redhat:ansible_tower:3"
    },
    {
      "product_name": "Red Hat Ceph Storage 2",
      "fix_state": "Not affected",
      "package_name": "ansible",
      "cpe": "cpe:/a:redhat:ceph_storage:2"
    },
    {
      "product_name": "Red Hat Ceph Storage 3",
      "fix_state": "Not affected",
      "package_name": "ansible",
      "cpe": "cpe:/a:redhat:ceph_storage:3"
    },
    {
      "product_name": "Red Hat OpenStack Platform 10 (Newton)",
      "fix_state": "Not affected",
      "package_name": "ansible",
      "cpe": "cpe:/a:redhat:openstack:10"
    },
    {
      "product_name": "Red Hat OpenStack Platform 13 (Queens)",
      "fix_state": "Not affected",
      "package_name": "ansible",
      "cpe": "cpe:/a:redhat:openstack:13"
    },
    {
      "product_name": "Red Hat Storage 3",
      "fix_state": "Not affected",
      "package_name": "ansible",
      "cpe": "cpe:/a:redhat:storage:3"
    }
  ],
  "threat_severity": "Moderate",
  "public_date": "2020-02-19T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2014-4678 ansible: safe_eval function does not properly restrict the code subset leads to arbitrary code execution via crafted instructions",
    "id": "1828684",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1828684"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "6.3",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:L/I:L/A:L",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-94",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2014-4678",
  "document_distribution": "",
  "details": [
    "The safe_eval function in Ansible before 1.6.4 does not properly restrict the code subset, which allows remote attackers to execute arbitrary code via crafted instructions. NOTE: this vulnerability exists because of an incomplete fix for CVE-2014-4657."
  ],
  "references": null
}